<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="prev_screen_button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Vissza az előző elemzésekhez</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_6">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>530</y>
     <width>961</width>
     <height>471</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="avg_vbox"/>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_7">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>961</width>
     <height>471</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="relation_vbox_1"/>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_8">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>530</y>
     <width>961</width>
     <height>471</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="relation_vbox_2"/>
  </widget>
  <widget class="QLabel" name="relation_label_1">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>20</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="relation_label_2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>20</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="avg_label">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>30</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="title_label">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>0</y>
     <width>1161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_9">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>60</y>
     <width>481</width>
     <height>471</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="relation_pos_vbox_1"/>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_10">
   <property name="geometry">
    <rect>
     <x>1440</x>
     <y>60</y>
     <width>481</width>
     <height>471</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="relation_pos_vbox_2"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
